.github/workflows/release.yml

```diff
@@ -68,82 +66,117 @@ jobs:
           sed -i -e "/HTTPS_VERSION/s/[0-9]*\.[0-9]*\.[0-9]*/"${HTTPS_VERSION}"/" .dappnode_profile
           cat .dappnode_profile
 
-      # ISO ATTENDED
-      - name: Build attended
+      - name: Build Debian attended
         run: |
+          sed -i -e "/BASE_OS/s/ubuntu/debian/" docker-compose.yml
           sed -i -e "/UNATTENDED/s/true/false/" docker-compose.yml 
-          docker-compose build
-          docker-compose up
+          docker-compose up --build
 
-      # Verify ISO attended created
-      - name: Check iso attended
+      - name: Check Debian ISO attended
         run: |
-          ls -lrt images/DAppNode-debian-bookworm-amd64.iso
+          ls -lrt images/Dappnode-debian-*.iso
 
-      # Set new name for the release asset
-      - name: Set DAppNode attended ISO name
+      - name: Set Debian Dappnode attended ISO name
         run: |
-          cp ./images/DAppNode-debian-bookworm-amd64.iso DAppNode-${CORE_VERSION}-debian-bookworm-amd64.iso
+          file=$(ls images/Dappnode-debian-*.iso)
+          attended_filename="${file/images\/Dappnode-/Dappnode-${CORE_VERSION}-}"
+          mv "$file" "$attended_filename"
```

Comment:
why this change everywhere? assuming cause its less resource expensive doing `mv` than `cp`?

Reply:
or is there any other reason?